deny.toml

```diff
@@ -8,6 +8,7 @@ feature-depth = 1
 
 [advisories]
 ignore = [
+    "RUSTSEC-2024-0375", # atty dependency being unmaintained, dependency of clap and criterion, we would need to update to newer major of dependencies
```

Comment:
nit: it's already in the list

Reply:
Ok, I'm not fixing this, as this PR blocks my colleague's work.